src/Containers/SparseAxisArray.jl

```diff
@@ -282,24 +282,25 @@ function Base.show(io::IOContext, x::SparseAxisArray)
     end
     limit = get(io, :limit, false)::Bool
     half_screen_rows = limit ? div(displaysize(io)[1] - 8, 2) : typemax(Int)
-    print_entry(i) = i < half_screen_rows || i > length(x) - half_screen_rows
-    # For stable printing, sort by the string value of the key.
-    key_strings = [(join(key, ", "), value) for (key, value) in x.data]
-    sort!(key_strings; by = x -> x[1])
-    pad = maximum(
-        print_entry(i) ? length(x[1]) : 0 for (i, x) in enumerate(key_strings)
-    )
     if !haskey(io, :compact)
         io = IOContext(io, :compact => true)
     end
+    # For stable printing, sort by the string value of the key.
```

Comment:
The printing is not stable anymore since the filter we apply depends of the order of the element in the dictionary iterators. We should comment about it to say that this is for performance reason